### PR TITLE
Added a fake poppy

### PR DESCRIPTION
**Pull request by foob1**
Added a fake poppy to trinkets under loadouts and the autodrobe

## About the PR
Added a fake poppy to trinkets under loadouts and the autodrobe, replacing the poppy in the autodrobe

## Why / Balance
Cosmetic purposes, and to make it so the poppy in the autodrobe is just for cosmetic purposes and can not be used as a healing, botany, or food item.

## Technical details
N/A

## Media

https://github.com/space-wizards/space-station-14/assets/134818173/0e36ead8-92e3-48f7-b304-19656be9e712

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
N/A

**Changelog**

:cl:
- add: Added fake poppy to loadouts and autodrobe
- remove: Removed poppy from autodrobe

### CODE REVIEW

**Comment by foob1** [comment]
Apologies for the video not capturing the full screen

**Comment by EmoGarbage404** [comment]
I dont really get why having a normal poppy in the drobe is a problem? Its kinda weird to make an identical looking fake flower to put in there (although tbh i dont know why its in there to begin with)

**Comment by foob1** [comment]
> I dont really get why having a normal poppy in the drobe is a problem? Its kinda weird to make an identical looking fake flower to put in there (although tbh i dont know why its in there to begin with)

The normal poppy in the drobe can be used as a food item, a healing item, and a botany item, and I believe it is the only item in the autodrobe that can be used for something other than a cosmetic purpose.

**Comment by lzk228** [comment]
> although tbh i dont know why its in there to begin with

it was replacement for hairflower
#25475

**Comment by Dutch-VanDerLinde** [comment]
> I dont really get why having a normal poppy in the drobe is a problem? Its kinda weird to make an identical looking fake flower to put in there (although tbh i dont know why its in there to begin with)

adding a fake poppy also allows it to be added to loadouts, for cosmetic purposes, because if you put the normal poppy in there people will just pick it so they can get free bicaridine flower

**Comment by FairlySadPanda** [comment]
Paper poppy? Then it can be eaten by moths like other clothing https://www.poppyshop.org.uk/products/paper-poppy-2

**Comment by foob1** [comment]
> Paper poppy? Then it can be eaten by moths like other clothing https://www.poppyshop.org.uk/products/paper-poppy-2

It can be eaten by moths yes

**Comment by Partmedia** [comment]
I don't feel that the upside of including an actual poppy are significant enough to warrant adding a fake poppy. I hear the concerns about keeping this item available for decoration and not having it raided, but what if we just add the ordinary poppy to the drobe and see if that actually becomes an issue?

**Comment by Dutch-VanDerLinde** [comment]
> I don't feel that the upside of including an actual poppy are significant enough to warrant adding a fake poppy. I hear the concerns about keeping this item available for decoration and not having it raided, but what if we just add the ordinary poppy to the drobe and see if that actually becomes an issue?

thats whats in the autodrobe already, the fake poppy replaces it

**Comment by foob1** [comment]
Trying to do #30079 instead of this, if it gets denied I'll reopen this one

**Comment by DieselMohawk** [comment]
Paper Poppies would have been a neat idea, I guess regular poppies will be fine as well though

**Comment by EdenTheLiznerd** [comment]
This would be awesome to have, fixes the powergaming issue and provides a fun cosmetic option, can't be classified as bloat either cause well... It's not even bloat.

**Comment by foob1** [comment]
> Trying to do #30079 instead of this, if it gets denied I'll reopen this one

Reopening the PR as I'm being told the poppy should be cosmetic

**Comment by IProduceWidgets** [comment]
Have you tried parenting a cosmeticpoppy off the regular poppy id and setting the solution to be empty?

**Comment by foob1** [comment]
> Have you tried parenting a cosmeticpoppy off the regular poppy id and setting the solution to be empty?

No I haven't (and I don't really know how I'm not that experienced), but wouldn't there still be the issue of it being able to be grown? The point of this PR and what the maintainers requested is to make the poppy in loadouts be purely cosmetic, without it being able to be used for anything else.

File: Resources/Prototypes/Entities/Clothing/Head/misc.yml
Comment (SlamBamActionman) [inline Resources/Prototypes/Entities/Clothing/Head/misc.yml:107]: You don't need to duplicate the poppy sprite to make it usable as clothing. 

1. Remove the fakepoppy.rsi sprites and folder.
2. Set the sprite path to that of the normal poppy sprite folder: https://github.com/space-wizards/space-station-14/tree/bb2981400cb34d476dd7f6f90aba4d0209dab047/Resources/Textures/Objects/Specific/Hydroponics/poppy.rsi
3. Add `state: produce` as a new line under the sprite component. This makes it use the `produce.png` sprite as the icon, which should be what you want.

You wouldn't need to add a state for the Clothing component though since that defaults to the file named `equipped-HELMET.png`

**Review by SlamBamActionman — CHANGES_REQUESTED** [review]
Making the fake poppy its own clothing object is perfectly fine and it shouldn't be parented to the original poppy since the only thing it shares is the visuals. Just make sure you remove the duplicate sprite folder and use the original ones, and I think this PR is good to go.

File: Resources/Prototypes/Loadouts/Miscellaneous/trinkets.yml
Comment (Dutch-VanDerLinde) [inline Resources/Prototypes/Loadouts/Miscellaneous/trinkets.yml:14]:
```suggestion
  storage:
    back:
    - ClothingHeadHatFakePoppy
```

File: Resources/Prototypes/Loadouts/Miscellaneous/trinkets.yml
Comment (Dutch-VanDerLinde) [inline Resources/Prototypes/Loadouts/Miscellaneous/trinkets.yml:16]:
```suggestion
```

File: Resources/Prototypes/Loadouts/Miscellaneous/trinkets.yml
Comment (Dutch-VanDerLinde) [inline Resources/Prototypes/Loadouts/Miscellaneous/trinkets.yml:20]:
```suggestion
```

File: Resources/Prototypes/Loadouts/Miscellaneous/trinkets.yml
Comment (Dutch-VanDerLinde) [inline Resources/Prototypes/Loadouts/Miscellaneous/trinkets.yml:19]:
```suggestion
```

File: Resources/Prototypes/Loadouts/Miscellaneous/trinkets.yml
Comment (Dutch-VanDerLinde) [inline Resources/Prototypes/Loadouts/Miscellaneous/trinkets.yml:18]:
```suggestion
```

File: Resources/Prototypes/Loadouts/Miscellaneous/trinkets.yml
Comment (Dutch-VanDerLinde) [inline Resources/Prototypes/Loadouts/Miscellaneous/trinkets.yml:21]:
```suggestion
```

File: Resources/Prototypes/Loadouts/Miscellaneous/trinkets.yml
Comment (Dutch-VanDerLinde) [inline Resources/Prototypes/Loadouts/Miscellaneous/trinkets.yml:17]:
```suggestion
```

**Review by Dutch-VanDerLinde — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Loadouts/Miscellaneous/trinkets.yml
Comment (Dutch-VanDerLinde) [inline Resources/Prototypes/Loadouts/Miscellaneous/trinkets.yml:21]:
```suggestion
# Decorations
- type: loadout
  id: ClothingHeadHatFakePoppy
  storage:
    back:
    - ClothingHeadHatFakePoppy
```

**Review by Dutch-VanDerLinde — CHANGES_REQUESTED** [review]

**Review by SlamBamActionman — APPROVED** [review]
LGTM

**Comment by foob1** [comment]
> How about making the fake poppy squirt water on someone upon you clicking with it on harm mode on them, like a water bottle? With a obvious squirt sound for increased comedic purposes. This way, it isn't just a 1:1 copy of a real poppy for no reason whatsoever. And of course, making the fake poppy a clown and mime exclusive starting item.

https://github.com/space-wizards/space-station-14/pull/30079#pullrequestreview-2190203551

**Comment by SlamBamActionman** [comment]
Hello! The PR was put up to a maintainer vote which concluded with the following result: Merge, but with changes.

Maintainers like the idea of a cosmetic flower in the loadout, but there were concerns about its item icon looking like a poppy and the name "fake poppy" not sounding so great. The PR has maintainer approval if the name is changed (just calling it "hairflower" was a popular suggestion) and the icon sprite is resprited to be visually distinct. The worn sprite can still remain the same since it's just 3x3 pixels.

**Comment by foob1** [comment]
Don't have motivation to contribute

**Pull request closed without merging**
